src/crypto/muhash.h

```diff
@@ -73,6 +74,13 @@ class MuHash3072
 
     /* Finalize into a 384-byte hash. Does not change this object's value. */
     void Finalize(unsigned char* hash384) noexcept;
+
+    SERIALIZE_METHODS(MuHash3072, obj)
+    {
+        for(int i = 0; i < obj.LIMBS; ++i) {
```

Comment:
d111135a3c928d1915f48cb4d46ca89a3d179686

nit, space after `for`.

Reply:
done